app/presenters/tree_builder_sections.rb

```diff
@@ -0,0 +1,79 @@
+class TreeBuilderSections < TreeBuilder
+  has_kids_for Hash, [:x_get_tree_hash_kids]
+
+  def initialize(name, type, sandbox, build = true, data, controller_name, current_tenant)
```

Comment:
there's no point in default values for an argument in the middle of the formal arg. list